com.unity.shadergraph/Editor/Drawing/Blackboard/BlackboardFieldPropertyView.cs

```diff
@@ -232,7 +232,7 @@ public BlackboardFieldPropertyView(AbstractMaterialGraph graph, IShaderProperty
                         return;
                     colorProperty.colorMode = (ColorMode)evt.newValue;
                     colorField.hdr = colorProperty.colorMode == ColorMode.HDR;
-                    colorField.DoRepaint();
```

Comment:
This should never be called directly since the state is not setup to do a repaint. 
It was public before and is becoming protected.